src/pages/task/TaskPageBody.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { Tab, TaskURLParams } from "pages/types/task";
+import { TestsTable } from "pages/task/TestsTable";
+import { FilesTables } from "pages/task/FilesTables";
+import { useParams } from "react-router-dom";
+
+export const TaskPageBody: React.FC = () => {
+  const { tab } = useParams<TaskURLParams>();
+  let BodyComp = <></>;
+  switch (tab) {
+    case Tab.Tests:
+      BodyComp = <TestsTable />;
+      break;
+    case Tab.Files:
+      BodyComp = <FilesTables />;
+      break;
+  }
+  return BodyComp;
+};
```

Comment:
i saw @john-m-liu's comment but this being a separate component adds unneeded complexity. a render function that returns tab component based on `useParams` would have been much more straight forward. because technically this is not really a component. it's conditional logic.

Reply:
taking a second look at this component and i really don't think it should be its own component right now. it's really simple and straight forward; therefore would be easier to read if all in the same file. when components begin to become complex is when it's time to refactor. premature refactoring components leads to unnecessary componentization. Leave things as they are until they begin to become complex.

Reply:
@tgrander I think this is a reasonable abstraction for now since this list will grow and the task page is already doing multiple things (making a request for the task, rerouting to default tab). We may potentially add additional logic to the Task page to render the tab labels.

Reply:
eh, sure. i still think a simple function could have done the trick. but why use `let` to reassign variables instead of just returning the component directly?